cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStore.java

```diff
@@ -481,88 +510,84 @@ private void addArchivesToList(List<ArtifactDetail> archives, Row row) throws IO
     }
   }
 
-  // given a row representing plugin metadata, add all plugins in the row to the given map
-  private void addPluginsToMap(Map<ArtifactInfo, List<PluginClass>> map, Row row) throws IOException {
-    // plugin key contains namespace, plugin type, and plugin name
-    PluginKey pluginKey = PluginKey.parse(row.getRow());
-
+  private void addPluginsToMap(Id.Artifact parentArtifactId, Map<ArtifactInfo, List<PluginClass>> map,
+                               Row row) throws IOException {
     // column is the artifact name and version, value is the serialized PluginClass
```

Comment:
Is this comment still true? I see namespace of the artifact also in the column in the javadoc for the class?